cmd/opampsupervisor/e2e_test.go

```diff
@@ -238,6 +250,62 @@ func TestSupervisorStartsCollectorWithRemoteConfig(t *testing.T) {
 	}, 10*time.Second, 500*time.Millisecond, "Log never appeared in output")
 }
 
+func TestSupervisorStartsCollectorWithNoOpAMPServer(t *testing.T) {
```

Comment:
Can we add a test for what happens if the Supervisor starts and there is both no OpAMP server and no prior config to start from? I realize in practice the Collector will be useless without some kind of config to use, but it would be nice to have a test that captures this scenario.

Reply:
Added a test for that scenario.